tests/test_barostat.py

```diff
@@ -606,9 +606,12 @@ def test_barostat_scaling_behavior():
     scaling = baro.get_volume_scale_factor()
     assert scaling > 0
 
+    # Set to an intentionally bad factor to ensure it adapts
+    bad_scaling_factor = 0.5 * compute_box_volume(host_box)
+    baro.set_volume_scale_factor(bad_scaling_factor)
     ctxt.multiple_steps(100)
     # The scaling should adapt between moves
-    assert scaling != baro.get_volume_scale_factor()
+    assert bad_scaling_factor != baro.get_volume_scale_factor()
```

Comment:
nit: could strengthen to:
(1) assert `baro.get_volume_scale_factor() == bad_scaling_factor` prior to `ctxt.multiple_steps`,
(2) assert `baro.get_volume_scale_factor() < bad_scaling_factor` (rather than just `!=`) after `ctxt.multiple_steps`

Reply:
Strengthened in https://github.com/proteneer/timemachine/pull/1159/commits/4f8b877ec86d7709e9039d5ef737b1a2b72de532